### ISSUE TITLE
kongregate.com: breakage

### ISSUE THREAD

**Issue by punisher018** [uBlockOrigin/uAssets#25144]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.kongregate.com/games/Magmic/passport-rummy-card-game`

### Category

breakage

### Description

`||mixpanel.com^$3p`  this filter is included in uBlock filters - Unbreak and EasyPrivacy. 

It blocks the `https://api.mixpanel.com/track/?ip=1` link, which causes the game to freeze later in the game, when the game ends or when you get the daily gift. While it was fine in the first games, it froze after I received the daily reward, and even if I refresh the page afterwards, the game froze continuously until I allowed this link. When I allowed this filter, I was able to play the game without any problems.

### Other extensions used

-

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.59.0
Chromium: 128
filterset (summary):
 network: 154992
 cosmetic: 58683
 scriptlet: 23961
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 4-0, never
  TUR-0: 13745-15, 14m
  easylist: 85822-574, 14m Δ
  easyprivacy: 53094-652, 14m Δ
  plowe-0: 3547-0, now
  ublock-badware: 10885-6, 14m Δ
  ublock-filters: 39790-350, 14m Δ
  ublock-privacy: 1119-2, 14m Δ
  ublock-quick-fixes: 242-0, 14m Δ
  ublock-unbreak: 2471-0, 14m Δ
  urlhaus-1: 28670-0, 14m
filterset (user): [array of 4 redacted]
userSettings:
 advancedUserEnabled: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 348 ms (selfie)
 maxAssetCacheWait: 70 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 24
 network:
  cpmstar.com: 2
  doubleclick.net: 1
  facebook.com: 1
  google-analytics.com: 1
  google.com: 7
  googletagmanager.com: 2
  imasdk.googleapis.com: 1
  mixpanel.com: 2
  newrelic.com: 2
  unity3d.com: 5
 extended:
  ##.square_ad
  ##.bg-gray-800.mx-4.p-1
  ##+js(trusted-replace-xhr-response, '/\{"brs_c[^\n]+?"SponsoredD…
  ##+js(trusted-replace-xhr-response, '/\{"cnt_lbl_brs"[^\n]+?"Spo…
  ##+js(json-prune, require.0.3.0.__bbox.require.[].3.1.__bbox.res…
  ##+js(trusted-replace-xhr-response, /\{"node":\{"role":"SEARCH_A…
  ##+js(trusted-replace-xhr-response, /\{"node":\{"__typename":"Ma…
  ##+js(trusted-replace-xhr-response, /\{"node":\{"__typename":"Vi…
  ##+js(json-prune-xhr-response, data.viewer.instream_video_ads da…
  ##+js(set-constant, rwt, noopFunc)
  ##+js(set-attr, c-wiz[data-p] [data-query] a[target="_blank"][ro…
```
</details>

**Comment by mapx-** [comment]
test

`@@||api.mixpanel.com/track/$xhr,domain=konggames.com`

**Comment by stephenhawk8054** [comment]
@punisher018 Are you logged in to the site?

**Comment by punisher018** [comment]
@mapx I was already able to continue the game by adding this filter, so yes it works. (I tested it by playing a lot)

@stephenhawk8054 Yes I was logged in, I didn't try by log out.

**Comment by stephenhawk8054** [comment]
@punisher018 Can you reproduce the issue when logging out?

**Comment by Yuki2718** [comment]
I see `https://api.mixpanel.com/track/?ip=1` blocked when I get rewards, but no freeze here.

**Comment by punisher018** [comment]
I played a little bit but I couldn't reproduce it. When I was playing yesterday the game froze, it froze when I refreshed the page, it froze when I entered the game again, I even reopened the game from a different browser and after a while it froze too. (when I got a reward at the end of the game or at the end of the game, spinning, etc.) It was fixed when I allowed the filter written above. I think it was broken when I won the new day reward (I coincided with the end of the server's 24 hours).
